zc_common/tests.py

```diff
@@ -0,0 +1,11 @@
+from django.test.runner import DiscoverRunner
```

Comment:
@tnsengimana We already have this  https://github.com/ZeroCater/zc_common/blob/master/zc_common/databaseless_test_runner.py